client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -39,3 +59,12 @@ export default connectOptions(
 		);
 	}
  );
+
+export default connect(
+	( state, { siteId } ) => {
+		const currentPlan = getCurrentPlan( state, siteId ) || null;
```

Comment:
Nothing in particular, just at some points it was "undefined" and at other points was "null", so I preferred to push it over the line.